src/components/LevelsModal.tsx

```diff
@@ -0,0 +1,154 @@
+          fill="#3D3D3D"/>
+        <path
+          d="M132.974 86.9095C132.926 87.0336 132.735 87.8444 132.554 88.7029C132.363 89.5519 132.058 90.8874 131.867 91.6601C131.686 92.4328 131.552 93.1101 131.591 93.1673C131.619 93.215 131.753 93.2627 131.877 93.2627C132.067 93.2627 132.172 93.1482 132.43 92.614C132.812 91.8604 133.298 90.4295 133.489 89.4947C133.661 88.6552 133.775 87.1576 133.689 86.8905C133.594 86.5852 133.098 86.5947 132.974 86.9095Z"
+          fill="#3D3D3D"/>
+        <path
+          d="M131.609 93.7099C131.456 93.8529 131.58 94.1964 132.277 95.4842C133.087 96.9818 135.243 101.379 136.264 103.612C137.075 105.367 137.142 105.491 137.39 105.453C137.79 105.395 137.8 105.157 137.466 104.07C136.483 100.836 134.833 97.6019 132.553 94.4253C132.038 93.7099 131.79 93.5286 131.609 93.7099Z"
+          fill="#3D3D3D"/>
+        <path
+          d="M94.9215 104.224C94.6448 104.405 94.9406 106.437 95.5034 108.192C95.8945 109.403 96.9056 111.426 97.6402 112.437C98.9852 114.326 101.093 115.957 101.818 115.68C102.477 115.432 102.171 114.65 100.693 112.837C98.8517 110.548 98.5369 110.138 98.0694 109.403C97.7928 108.965 97.0773 107.658 96.4764 106.503C95.1695 103.985 95.2172 104.042 94.9215 104.224Z"
+          fill="#3D3D3D"/>
+      </svg>
+      </div>
+    )
+  }
+}
+
+function BackgroundPaperSvg() {
```

Comment:
Isn't this already a component somewhere? Should make an own file for this and the other svg components.